ovos_solver_aiml_plugin/__init__.py

```
@@ -94,31 +95,29 @@ def shutdown(self):
-            config["lang"] = lang = "en-us"
-        super().__init__(config, internal_lang=lang, enable_tx=True, priority = 95)
-        self.brain = AimlBot(lang)
+            config["lang"] = "en-us"
+        super().__init__(config)
+        self.brain = AimlBot(self.config.get("lang"))
         self.brain.load_brain()
 
-    def get_spoken_answer(self, query: str,
-                          lang: Optional[str] = None,
-                          units: Optional[str] = None) -> Optional[str]:
+    def query(self, query: str, lang: Optional[str] = None, k: int = 3) -> List[Tuple[str, float]]:
```

Ruff (0.15.0)

[warning] Unused method argument: `lang`

(ARG002)